### DISCUSSION TITLE
Docker agent configuration reference

### DISCUSSION THREAD

**Discussion by marcusthoreson-UG** [dagster-io/dagster#18983] [Q&A] [answered]
I'm setting up a code location within a Docker agent, and I wanted to configure some runtime settings for the location's container.
There are a couple of places ([here](https://github.com/dagster-io/dagster/discussions/12621#discussioncomment-5168873) and [here](https://github.com/dagster-io/dagster/discussions/14771#discussioncomment-6197823)) in the Dagster github discussions that seem to reference a `container_context.docker.container_kwargs` property being used in the `dagster_cloud.yaml` file, but that property is missing from the [latest docs](https://docs.dagster.io/dagster-cloud/deployment/agents/docker/configuration-reference).

Is this an experimental feature, an undocumented one, or does it actually not exist?

**Comment by gibsondan** [comment, marked as answer]
You can indeed set container_kwargs in a dagster_cloud.yaml file, like the following:

```
container_context:
  docker:
    container_kwargs:
      auto_remove: true
```

That is missing from the Docker agent docs but we'll get that fixed, thanks for the report!
